installer/README.md

```diff
@@ -458,6 +458,22 @@ Kubernetes objects, such as your TLS certificate or connection secrets.
 kubectl create namespace gitpod
 ```
 
+# I need to add additional configuration to my load balancer - what do I do?
+
+By default, this will create a `proxy` service with `ServiceType: LoadBalancer`.
+If you want to specify your own ingress, you can do this easily by changing
+your Installer configuration:
+
+```yaml
+components:
+  proxy:
+    serviceType: ClusterIP
```

Comment:
Maybe add a validation checking that `serviceType` is LoadBalancer or ClusterIP only?
Both, `proxy` and `ws-proxy` are configured on ports 80 and 443